modulemd/v2/meson.build

```
@@ -626,6 +626,45 @@ test ('rpmmap_python2_release', python2,
       env : py_test_release_env,
       args : rpmmap_python_script)
 
+# -- Test Modulemd.TranslationHelpers (Python) -- #
+
+translation_helpers_python_script = files('../../ModulemdTranslationHelpers/Utils.py')
+get_babel = '''
+import babel
+'''
+
+# Python 2
+if (get_option('with_py2_overrides'))
+  ret2 = run_command([python2, '-c', get_babel])
+
+  if ret2.returncode() != 0
+    error('Failed to import Python 2 babel')
```

Here's the problem. The `error()` directive in meson means "Print this error and then quit with a non-zero exit code". You need to store a variable and just use `message()` to print appropriate text instead. Something like:
```suggestion
    warning('No Babel on Python 2: skipping TranslationHelper tests')
```